todolist/app/screens/ListBook/index.js

```diff
@@ -2,17 +2,15 @@ import React from 'react';
 import { View } from 'react-native';
 
 import styles from './styles';
-import Title from '../../components/Title';
 import List from '../../components/List';
 import Book from '../../components/Book';
 
 import books from '../../constants/listBoook';
 
-function BookList() {
+function BookList({ navigation }) {
   return (
     <View style={styles.container}>
-      <Title value={'Book List'}/>
-      <List ComponentChild={Book} items={books}/>
+      <List ComponentChild={Book} items={books} navigate={navigation.navigate} />
```

Comment:
mmm i'm not convinced.. you should not need to pass navigation to the List. You can use NavigationActions directly from react-navigation:
`import { NavigationActions } from 'react-navigation';`
And then you can use it the same way that you are using navigate prop:
`dispatch(NavigationActions.navigate({ routeName: Routes.RecoverPassword }));`